lib/server_sent_events/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ServerSentEvents
-  VERSION = "0.1.2"
+  VERSION = "0.1.3"
```

Comment:
Version number bump has nothing to do with the Ruby version constraint relaxation. This should be in a separate commit or even entirely absent from this PR.

Reply:
Gotcha, removed it.